### PR TITLE
chore: Make the `range_proof_commitment` field of the `SecretKeyShareEncryptionAndProof` struct public

### PR DESCRIPTION
**Pull request by ItayLevyOfficial**

### CODE REVIEW

**Review by zeevmoney — APPROVED** [review]
@ycscaly

File: src/dkg/centralized_party/decommitment_round.rs [hunk 1]
```diff
@@ -180,16 +180,16 @@ where
...
-                decentralized_party_public_key_share.clone(),
-            )
-                .into(),
+        // let a = (
+        //     encrypted_decentralized_party_secret_key_share.clone(),
+        //     decentralized_party_public_key_share.clone(),
+        // )
+        let a = (
+            encrypted_decentralized_party_secret_key_share.clone(),
+            decentralized_party_public_key_share.clone(),
```
Comment (ycscaly) [inline src/dkg/centralized_party/decommitment_round.rs:189]: what is this? how can a code like this be included in a pr?

File: src/dkg/decentralized_party.rs [hunk 2]
```diff
@@ -23,7 +23,7 @@ pub struct SecretKeyShareEncryptionAndProof<
 > {
     pub(in crate::dkg) public_key_share: GroupElementValue,
     pub(in crate::dkg) encrypted_secret_key_share: CiphertextValue,
-    pub(in crate::dkg) range_proof_commitment: RangeProofCommitmentValue,
+    pub range_proof_commitment: RangeProofCommitmentValue,
```
Comment (ycscaly) [inline src/dkg/decentralized_party.rs:26]: no no no, this range proof commitment is one-time use, for a single proof.
This is definitely not where you should take it from - doing so will break the proof

File: src/dkg/centralized_party/decommitment_round.rs [hunk 3]
```diff
@@ -26,7 +26,7 @@ pub struct Output<GroupElementValue, ScalarValue, CiphertextSpaceValue> {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
 pub struct PublicKeyShareDecommitmentAndProof<GroupElementValue, DLProof> {
     pub(in crate::dkg) proof: DLProof,
-    pub(in crate::dkg) public_key_share: GroupElementValue,
+    pub public_key_share: GroupElementValue,
```
Comment (ycscaly) [inline src/dkg/centralized_party/decommitment_round.rs:29]: This is not where you should take it from. You should take it from the [DKG output](https://github.com/dwallet-labs/2pc-mpc/blob/c509064e18f5261ec298d1d4d697dfb260ed85fe/src/dkg/decentralized_party/decommitment_proof_verification_round.rs#L29).

**Review by ycscaly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**